### PR TITLE
Add support for specifying the test name without the category prefix.

### PR DESCRIPTION
**Pull request by hellais**
This implements: https://github.com/TheTorProject/ooni-probe/issues/483

### CODE REVIEW

**Comment by bassosimone** [comment]
I am currently testing this!

File: ooni/deck.py [hunk 1]
```diff
@@ -85,11 +88,30 @@ def nettest_to_path(path, allow_arbitrary_paths=False):
...
         return path
 
-    fp = FilePath(config.nettest_directory).preauthChild(path + '.py')
-    if fp.exists():
-        return fp.path
-    else:
+    p = path.split("/")
+    test_name = p[0]
+    if len(p) > 1:
+        test_categories = [p[0]]
```
Comment (bassosimone) [inline ooni/deck.py]: This assignment is redundant because below you assign unconditionally to `test_categories`

File: ooni/deck.py [hunk 2]
```diff
@@ -85,11 +88,30 @@ def nettest_to_path(path, allow_arbitrary_paths=False):
     if allow_arbitrary_paths and os.path.exists(path):
         return path
 
-    fp = FilePath(config.nettest_directory).preauthChild(path + '.py')
-    if fp.exists():
-        return fp.path
-    else:
+    p = path.split("/")
```
Comment (bassosimone) [inline ooni/deck.py]: Would it make sense here to use `path.rsplit("/", 1)`?

**Comment by bassosimone** [comment]
- [ ] testing
- [ ] diff review
- [ ] fixes
- [ ] merging!

File: ooni/deck.py (hunk 2 above)
Comment (hellais) [inline ooni/deck.py]: Fair point. I guess we could do an `rsplit` and get the last item as the test_name and ignore the test_category section.

**Comment by bassosimone** [comment]
Tested again after d78e7d4, it works! Merging!

**Pull request merged**